src/protocols/hii_font.rs

```diff
@@ -0,0 +1,188 @@
+//! HII (Human Interface Infrastructure) Font Protocol and Font Protocol Ex from
+//! Sections 34.1 and 34.2
+//!
+//! Interfaces which retrieve font information.
+
+pub const PROTOCOL_GUID: crate::base::Guid = crate::base::Guid::from_fields(
+    0xe9ca4775,
+    0x8657,
+    0x47fc,
+    0x97,
+    0xe7,
+    &[0x7e, 0xd6, 0x5a, 0x8, 0x43, 0x24],
+);
+
+pub const EX_PROTOCOL_GUID: crate::base::Guid = crate::base::Guid::from_fields(
```

Comment:
@dvdhrm 
I like how you've cleaned up the namespaces elsewhere in the tree, and this was a stab at continuing that. Happy to move this to  its own module, if desired.

Reply:
I split them apart again, mainly because it makes it less controversial. So far, all the protocols are exported verbatim in `crate::protocols::`, and if we don't deviate, there is no reason for anyone to complain.

However, I do see that this makes the names diverge from the specification. Initially, I intended the `efi::` namespace to contain all the imports verbatim, so `EFI_FOO_BAR` from the spec would translate to `efi::FooBar` (or `efi::foo_bar` fwiw). This might be a bit cumbersome, though. So if you are interested in keeping names closer to the spec, we could make use of the fact that the EFI spec has a flat namespace, and thus conflicts with our protocol-namespace are quite unlikely (though still possible..). So we could have a soft rule that we export aliases from protocols, if they match the naming in the specification. For example:

The spec calls it `EFI_FONT_DISPLAY_INFO` but defines it in `EFI_FONT_EX`. Hence, we could add:

```rust
mod protocols::hii_font { // i.e. in hii_font.rs
    pub use super::hii_font_ex::DisplayInfo;
}
```

I would be open to that, as it is an easy rule to follow and it would produce more predictable names. If you have a strong reason to merge the protocols, I can undo the split as well. Comments welcome!